src/main/java/gov/cms/ab2d/worker/processor/PatientClaimsProcessorImpl.java

```diff
@@ -0,0 +1,179 @@
+import gov.cms.ab2d.fetcher.model.JobFetchPayload;
+import gov.cms.ab2d.fhir.BundleUtils;
+import gov.cms.ab2d.fhir.FhirVersion;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.hl7.fhir.instance.model.api.IBaseBundle;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.OffsetDateTime;
+import java.util.List;
+
+import static gov.cms.ab2d.aggregator.FileOutputType.DATA;
+import static gov.cms.ab2d.aggregator.FileOutputType.ERROR;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class PatientClaimsProcessorImpl {
+
+    private final BFDClient bfdClient;
+    
+    private final EventClient eventClient;
+
+    @Value("${efs.mount}")
+    private final String efsMount;
+
+    public void writeOutData(JobFetchPayload jobFetchPayload, ProgressTrackerUpdate update) throws IOException {
+        File file = null;
+        try (ClaimsStream stream = buildClaimStream(jobFetchPayload.getJobId(), DATA)) {
+            file = stream.getFile();
+            eventClient.send(new FileEvent(jobFetchPayload.getOrganization(), jobFetchPayload.getJobId(),
+                    stream.getFile(), FileEvent.FileStatus.OPEN));
+            for (EOBFetchParams fetchParams : jobFetchPayload.buildParams()) {
+                List<IBaseResource> eobs = getEobBundleResources(fetchParams);
+                writeOutResource(jobFetchPayload, fetchParams.getVersion(), update, eobs, stream);
+                update.incPatientProcessCount();
+            }
+        } finally {
+            eventClient.send(new FileEvent(jobFetchPayload.getOrganization(), jobFetchPayload.getJobId(),
+                    file, FileEvent.FileStatus.CLOSE));
+        }
+    }
+
+    void writeOutErrors(String anyErrors, JobFetchPayload jobFetchPayload) {
+        File errorFile = null;
+        try (ClaimsStream stream = buildClaimStream(jobFetchPayload.getJobId(), ERROR)) {
+            errorFile = stream.getFile();
+            eventClient.send(new FileEvent(jobFetchPayload.getOrganization(), jobFetchPayload.getJobId(),
+                    stream.getFile(), FileEvent.FileStatus.OPEN));
+            stream.write(anyErrors);
+        } catch (IOException e) {
+            log.error("Cannot log error to error file");
+        } finally {
+            eventClient.send(new FileEvent(jobFetchPayload.getOrganization(), jobFetchPayload.getJobId(),
+                    errorFile, FileEvent.FileStatus.CLOSE));
+        }
+    }
+
+    private ClaimsStream buildClaimStream(String jobId, FileOutputType outputType) throws IOException {
+        return new ClaimsStream(jobId, efsMount, outputType,
+                "streaming", "finished", (int) FileUtils.ONE_MB);
+    }
```

Comment:
It's a convention.  Otherwise, let's create a common constant that is shared across projects.  It is something that we never change, so making it a property is overkill in complexity.